pkg/cmd/server/admin/create_kubeconfig.go

```diff
@@ -183,6 +186,7 @@ func (o CreateKubeConfigOptions) CreateKubeConfig() (*clientcmdapi.Config, error
 		CurrentContext: o.ContextNick,
 	}
 
+	fmt.Fprintf(o.GetOut(), "Generating '%s' API client config as %s\n", o.UserNick, o.KubeConfigFile)
```

Comment:
I meant this command in particular.  You have the filename directly in this method.  Check before doing work.